packages/instant/src/redux/actions.ts

```diff
@@ -45,4 +46,5 @@ export const actions = {
     setError: (error?: any) => createAction(ActionTypes.SET_ERROR, error),
     hideError: () => createAction(ActionTypes.HIDE_ERROR),
     clearError: () => createAction(ActionTypes.CLEAR_ERROR),
+    clearBuyQuoteAndSelectedAssetAmount: () => createAction(ActionTypes.CLEAR_BUY_QUOTE_AND_SELECTED_ASSET_AMOUNT),
```

Comment:
I think it's ok to name these actions & action creators a bit more like an API... so this would be `resetSomethingSomething?` Just a thought.